### ISSUE TITLE
Icon request: repl (repl-it replit)

### ISSUE THREAD

**Issue by llamax1** [FortAwesome/Font-Awesome#14758]
### What can it be used for?

[REPL](repl.it) is a popular code-sharing platform and online IDE. The request is for a repl.it social media icon.

### Example image (optional)

![](https://repl.it/public/images/apple-touch-icon.png)

### Icon request checklist

- [x] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x] I have included at least one example image if it could be helpful (optional)
- [x] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by powerfulsheron** [comment]
+1

**Comment by lego3708** [comment]
+1

**Comment by Purpzie** [comment]
I was about to make a request for this until I just found this one. +1

**Comment by Purpzie** [comment]
Update: Repl.it now provides benefits in the [GitHub student developer pack.](https://education.github.com/pack), so its logo is probably needed even more now in FontAwesome.

**Comment by kenoldavid** [comment]
Any idea on when this will be implemented/available?

**Comment by llamax1** [comment]
@kenoldavid I don't know. There is more information at the bottom of [this page](https://fontawesome.com/community/leaderboard). The only thing I can recommend to push this idea is to get everyone you know with a GitHub account to thumbs-up this issue and move it to the top of the leaderboard.

**Comment by UmairJibran** [comment]
I came to post this issue, and then I stumbled upon here, thanks for making this request, hopefully, @davegandy or @robmadole or @supercodepoet or @tagliala or @talbs will see this, and provide us with one.

**Comment by Codemonkey51** [comment]
+1

**Comment by gellowg** [comment]
+1. Repl.it can be used as a portfolio or code sharing platform. There is so many use cases of this icon.

**Comment by wgyt** [comment]
I found an official svg version of the logo that could be used:
![](https://repl.it/public/images/logo.svg)

**Comment by llamax1** [comment]
> +1. Repl.it can be used as a portfolio or code sharing platform. There is so many use cases of this icon.

@gellowg Exactly! My inspiration for making this icon request was when I was making a personal website and wanted to link to my Repl profile

> I found an official svg version of the logo that could be used:
> ![](https://camo.githubusercontent.com/65dc590a48b6474abb0f4ad00aaef37ad2797ed813b52b0e7cbc87640032df56/68747470733a2f2f7265706c2e69742f7075626c69632f696d616765732f6c6f676f2e737667)

@wgyt Thank you. Hopefully this will encourage them...

**Comment by wsgdandy** [comment]
+1

**Comment by benfwalla** [comment]
+1 on this!

But, while it's not there, I created one myself for [my personal site](https://replit.com/@bewal416/me) (on Replit, of course).

1. I wanted the Replit logo to be white with an orange background, so I took that .svg from @williamhorning post above and [edited it to white](https://replit.com/@bewal416/me#assets/images/replit-white.svg).
2. I then played around with [the css](https://replit.com/@bewal416/me#assets/css/styles.css) by creating a class called `replit` to put Replit's official orange (#f26208) behind it as a background and properly center it. My site was built on a Boostrap template, so you may have to do your own hacking to stylize it the way you want.
3. The output is a circular, orange icon that looks just like the Fontawesome icons next to it!

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
